frontend/src/components/pages/PlayPage.js

```diff
@@ -35,22 +37,24 @@ export default function PlayPage() {
     </PlayPageStyled>
   );
   function getPlayerData() {
-    return game.players.find((player) => player.id === userData.playerId);
+    return game.betSession.players.find(
+      (player) => player.id === userData.playerId
+    );
   }
   function calcMinBet() {
     return (
-      Math.max(...game.players.map((player) => player.currentBet)) -
+      Math.max(...game.betSession.players.map((player) => player.currentBet)) -
       getPlayerData().currentBet
     );
   }
   function isActive(id) {
-    return id === game.players[game.activePlayerIndex].id;
+    return id === game.betSession.players[game.betSession.activePlayerIndex].id;
   }
 
   function calcPot() {
-    return game.players
+    return game.betSession.players
       .map((player) => player.currentBet)
-      .reduce((sum, bet) => sum + bet);
+      .reduce((sum, currentBet) => sum + currentBet);
```

Comment:
Realize that `reduce` iterates over the all array elements and you can do the mapping in there

```
return game.betSession.players
      .reduce((sum, player) => sum + player.currentBet, 0);
```